### PR TITLE
Feature/iohkle 179 build variants

### PR DESCRIPTION
**Pull request by StaceC**
Once the Custom CA is setup, `make load` and `make test` should now build the separate interfaces without requiring user intervention to load. Delete is the same.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,5 +79,48 @@ To set up your environment on Mac, using [Vagrant](https://www.vagrantup.com) is
...
+
 [1] *Note that this is because Docker for Mac does not support USB connectivity due to [xhyve limitations](https://github.com/mist64/xhyve#what-is-bhyve)*
+
+## Deploying
+
+The build process is managed with [Make](https://www.gnu.org/software/make/).
+
+### Setting Up Custom CA for signing Ledger App
+
+To remove the prompts when installing, running and deleting the app from the Ledger device, you need to generate a Custom CA keypair.
```
Comment (DCRichards) [inline README.md:91]: Should we write a `ca.sh` script to do this? Make life a bit faster?

**Review by DCRichards — APPROVED** [review]
Awesome work!

File: README.md (hunk 1 above)
Comment (StaceC) [reply README.md:91]: For sure. But atm it only needs to be done once, so not too painful. Plus, I get the sense some awk would be needed as the output from the genCAPair comes in two lines - 1st publick key, 2nd private key. I'll add a story to the backlog. Be nice to have it as part of the vagrant provisioning script...

Comment (DCRichards) [reply README.md:91]: Now we're talking!! I'm happy for this to be merged, regain some significant time from my life!

**Pull request merged**